test/tests/functormaterials/parsed_functor_material/parsed_functor_material.i

```diff
@@ -0,0 +1,59 @@
+[Mesh]
+  type = GeneratedMesh
+  dim = 3
+  nx = 2
+  ny = 2
+  nz = 2
+  xmin = 0.0
+  xmax = 4.0
+  ymin = 0.0
+  ymax = 6.0
+  zmin = 0.0
+  zmax = 10.0
+[]
+
+[Functions]
+  [fn1]
+    type = ConstantFunction
+    value = 5
+  []
+  [fn2]
+    type = ConstantFunction
```

Comment:
Could we do something a little more exotic than a constant function, just to be triply sure that the space and time arguments are getting properly forwarded to the functors used by the parsed functor?